cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/JarResolverImpl.java

```diff
@@ -124,6 +128,10 @@ public Path moduleTestJar(BLangPackage bLangPackage) {
         List<Path> modulePlatformLibs = new ArrayList<>();
         // copy platform libs for all modules(imported modules as well)
         addPlatformLibs(packageID, modulePlatformLibs);
+        // copy choreo extension if --observability-included flag is added
+        if (!isChoreoExtensionSkipped) {
+            modulePlatformLibs.add(getChoreoRuntimeJar());
+        }
```

Comment:
Can we add a test for this ? we can write an integration test to see of the observability classes are packed in to the final jar